src/lib/cache.ts

```diff
@@ -27,12 +27,13 @@ const VerboseEvents = [
 ]
 
 const uncompressedKeyPrefix = "::"
+const cacheVersion = "v1"
 
 export const cacheKey = key => {
   if (CACHE_COMPRESSION_DISABLED) {
     return uncompressedKeyPrefix + key
   } else {
-    return key
+    return cacheVersion + key
```

Comment:
We should check into default memcached behavior when it comes to evicting keys, as I am pretty sure our cache is full with the current 30 day key TTL.  If it chooses to evict keys based on shortest TTL it may end up evicting these new keys before their TTL expires.

Reply:
Effectively this change in cache key means we will miss all cached keys, so is there any reason not to flush the entire cache?

Reply:
My understanding was that memcached evicts least-recently-used items first, so the previous generation of cached values should be evicted before any of these new values. I think this is exactly the behavior we want, and since it's been necessary on a few past occasions to revise the cache key scheme like this, we figured prepending a version would be helpful addition.